.github/workflows/reusable-build-test-release.yml

```diff
@@ -33,7 +39,7 @@ on:
         required: false
         description: "branch for k8s manifests to run the tests on"
         type: string
-        default: "v3.1.0"
+        default: "feat/run-both-ucc-and-legacy-modinput-tests"
```

Comment:
Let's merge this PR https://github.com/splunk/ta-automation-k8s-manifests/pull/110#pullrequestreview-2469961146 at first and put here new version

Reply:
done!